src/modules/cli/hooks/useCLI.ts

```diff
@@ -48,22 +67,43 @@ const useCLI = (vault: UseVaultDetailsReturn['vault']) => {
       },
     },
     {
-      label: 'Add other tokens',
+      label: CLIFeaturesLabels.ADD_OTHER_TOKENS,
       icon: BitCoinIcon,
-      disabled: true,
-      onClick: () => {},
+      disabled: !hasPermission,
+      onClick: () => {
+        handleOpen(features[CLIFeaturesLabels.ADD_OTHER_TOKENS]);
+        createGTMCustomEvent({
```

Comment:
you created this static value in the form hook, why don't you use it here?

file: `src/modules/cli/hooks/CommingSoon/useCommingSoon.ts`

on hook, create a function to emmit event, and on this form, call fn pass by parameter the field enum `CLIFeaturesLabels.RECOVERY`